src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $main-text-color;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  background: url(../images/imac.jpeg);
```

Comment:
it's a bad practice to set the image as background of card, better to rewrite it using the img in the next task